### PR TITLE
Ft/rename google analytics extension

### PR DESCRIPTION
**Pull request by marketingKatimas**
Google Analytics Extension, spelling mistake and update the URL

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -492,9 +516,9 @@ <h3 id="textFindUs">Find Us</h3>
     <!-- Language-switch JS -->
     <!-- <script src="assets/js/language-switch.js"></script> -->
```
Comment (83clong) [inline index.html:517]: Is this still needed?

File: about-us.html [hunk 2]
```diff
@@ -342,8 +366,8 @@ <h3 id="textFindUs">Find Us</h3>
         <script src="assets/js/custom.js"></script>
         <!-- Language-switch JS -->
         <script src="assets/js/language-switch-about.js"></script>
-        
-        
+
+
```
Comment (83clong) [inline about-us.html:370]: remove the empty

File: index.html [hunk 3]
```diff
@@ -492,9 +516,9 @@ <h3 id="textFindUs">Find Us</h3>
...
     <script src="assets/js/language-switch-home.js"></script>
-    
 
-    
+
+
```
Comment (83clong) [inline index.html:521]: remove the empty

**Review by 83clong — APPROVED** [review]
After the changes, you can push this to master

File: index.html (hunk 1 above)
Comment (marketingKatimas) [reply index.html]: Done, remove.

File: about-us.html (hunk 2 above)
Comment (marketingKatimas) [reply about-us.html]: Done, remove

File: index.html (hunk 3 above)
Comment (marketingKatimas) [reply index.html]: Done, remove

**Pull request merged**